sdk/core/core-tracing/test/cache.spec.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import assert from "assert";
+import * as fs from "fs";
+import * as path from "path";
+
+const validOpenTelemetryVersions = ["1.0.0-rc1", "0.20.0"];
```

Comment:
I just noticed that it looks like 1.0 has shipped: https://www.npmjs.com/package/@opentelemetry/api/v/1.0.0

Should we just update to that and GA this package instead? :)

Reply:
Oh wow! Oh wow! Oh wow!

Reply:
But 1.0.0 has the next tag... and 0.20.0 has the latest tag